src/ol_openedx_course_export/LICENSE.txt

```diff
@@ -0,0 +1,672 @@
+
+                    GNU AFFERO GENERAL PUBLIC LICENSE
```

Comment:
We typically use the BSD-3-Clause license as it is compatible with the GPL/AGPL without being as restrictive. That should already be present in the root of the repository.

Reply:
Updated, I've also changed the copyright to use 2022 instead of 2021.

In addition, our other plugins like `ol-openedx-canvas-integration` and `ol-openedx-rapid-response-reports` are also using GNU licenses which I think should be updated in another PR along with copyright 2022 in the root's `license.txt`.